portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Joey's Image Gallery</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h1>Image Gallery</h1>
+      <p>Click <a href="index.html">here</a> to see return home.</p>
+      <p>Here are some images of my life at the moment:</p>
+      <p></p>
+
+      <p>Here is me</p>
+      <img class="images" src="/images/me.jpeg">
+      <p>I enjoy the outdoors. This is at Lake Tahoe!</p>
+      <img class="images" src="/images/Tahoe.JPG">
+      <p>During quarantine, my dad decided we were going to remodel the kitchen</p>
+      <img class="images" src="/images/Construction1.JPG">
+      <p>We continued with demolition</p>
+      <img class="images" src="/images/Construction2.JPG">
+      <p>Finally done with demolition</p>
+      <img class="images" src="/images/Construction3.JPG">
+      <p>We put in lights and some of the new cabinets</p>
+      <img class="images" src="/images/Construction4.JPG">
```

Comment:
Recommend that you standardize file names (namely captilizaing format) and file extension

Reply:
Do you mean changing "Construction1.JPG" to "construction1.jpg"?